web/src/routes/RepositoryDetails/Builds/BuildHistoryStartBuildModal.tsx

```diff
@@ -0,0 +1,118 @@
+import {
+  Modal,
+  ModalVariant,
+  Button,
+  Title,
+  Tabs,
+  Tab,
+  TabTitleText,
+  Tooltip,
+} from '@patternfly/react-core';
+import {Table, Tbody, Td, Th, Thead, Tr} from '@patternfly/react-table';
+import {useState} from 'react';
+import {RepositoryBuildTrigger} from 'src/resources/BuildResource';
+import BuildTriggerDescription from './BuildTriggerDescription';
+import Conditional from 'src/components/empty/Conditional';
+import DockerfileUploadBuild from './BuildHistoryStartBuildModalUploadDockerfile';
+
+export default function StartBuildModal(props: StartNewBuildModalProps) {
+  const [activeTabKey, setActiveTabKey] = useState<string | number>(0);
+  return (
+    <Modal
+      id="start-build-modal"
+      isOpen={props.isOpen}
+      onClose={() => props.onClose()}
+      variant={ModalVariant.medium}
+      style={{
+        overflowX: 'visible',
+        overflowY: 'visible',
+      }}
+    >
+      <Title headingLevel="h4">Start Repository Build</Title>
+      <Tabs
+        activeKey={activeTabKey}
+        onSelect={(_, eventKey) => setActiveTabKey(eventKey)}
+        aria-label="Start a new build tabs"
+        role="region"
+      >
+        <Tab
+          eventKey={0}
+          title={<TabTitleText>Invoke Build Trigger</TabTitleText>}
+          aria-label="invoke build trigger tab"
+        >
+          <p>
+            Manually running a build trigger provides the means for invoking a
+            build trigger as-if called from the underlying service for the
+            latest commit to a particular branch or tag.
+          </p>
+          <Table variant="compact">
+            <Thead>
+              <Tr>
+                <Th>Trigger Description</Th>
+                <Th>Branches/Tags</Th>
+                <Th></Th>
+              </Tr>
+            </Thead>
+            <Tbody>
+              <Conditional if={props.triggers?.length === 0}>
+                <Tr>
+                  <Td colSpan={3}>
+                    <p>No build triggers available for this repository.</p>
+                  </Td>
+                </Tr>
+              </Conditional>
+              {props.triggers.map((trigger) => (
+                <Tr key={trigger.id}>
+                  <Td>
+                    <BuildTriggerDescription trigger={trigger} />
+                  </Td>
+                  <Td>{trigger?.config?.branchtag_regex || 'All'}</Td>
+                  <Td>
+                    <Conditional if={trigger.can_invoke}>
+                      <a onClick={() => props.onSelectTrigger(trigger)}>
+                        Run Trigger Now
```

Comment:
Fixed, i removed the option

Reply:
how about simply greying out the option instead of removing the kebab option so that the user sees it and knows its disabled? We could use the `isDisabled` prop on [DropdownItem](https://www.patternfly.org/components/menus/application-launcher/#dropdownitem)

Reply:
i agree, that's probably a better UX